internal/db/code.go

```diff
@@ -18,7 +18,9 @@ type Code struct {
 type FunctionSignature int64
 
 const (
-	FromJSON FunctionSignature = iota
+	//Making the default not zero, so that the starlark runtime executing
```

Comment:
I'm not quite sure I follow this comment, but maybe you could just make Invalid = iota the first entry here?